src/db/spanner/batch.rs

```diff
@@ -181,39 +185,38 @@ pub async fn commit_async(
         // supplied in this batch
         let mut timer2 = db.metrics.clone();
         timer2.start_timer("storage.spanner.apply_batch_update", None);
+        let (sqlparams, mut sqlparam_types) = params! {
+            "fxa_uid" => params.user_id.fxa_uid.clone(),
+            "fxa_kid" => params.user_id.fxa_kid.clone(),
+            "collection_id" => collection_id,
+            "batch_id" => params.batch.id.clone(),
+            "timestamp" => as_rfc3339.clone(),
+        };
+        sqlparam_types.insert("timestamp".to_owned(), as_type(TypeCode::TIMESTAMP));
```

Comment:
NOT THAT WE NEED TO DO THIS NOW

But we really could take advantage of they SyncTimestamp type better for stuff like this. 
(heh, Good first bug material.)